workspaces/orchestrator/plugins/orchestrator-backend/src/service/api/mapping/V2Mappings.ts

```diff
@@ -142,6 +142,7 @@ export function mapToProcessInstanceDTO(
     duration: duration,
     // @ts-ignore
     workflowdata: variables?.workflowdata,
+    assessmentInstanceId: variables?.orchestratorAssessmentInstanceId as string,
```

Comment:
where is `assessmentInstanceId` used in the front-end? I am asking as now the request will return this new variable instead of `businessKey` but I do not see any changes on the FE plugin

Reply:
this is why it is a draft PR. we need to wait for sonata to support it in official release and then someone will have to change the FE